modules/acs-architecture-overview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_module-type: REFERENCE
+[id="acs-architecture_{context}"]
+= {product-title} architecture
+
+{product-title} installs as a pod in your OpenShift or a Kubernetes cluster and it includes the following components:
+
+|===
+| Component | Description | Quantity
+
+| <<central,*Central*>>
+| Gathers and displays information from other components.
+| *1* for multiple clusters.
+
+| <<sensor,*Sensor*>>
+| Collects and augments data from the Collector.
+| *1* for each cluster.
+
+| <<scanner,*Scanner*>>
+| Scans images for vulnerabilities.
+| *1* for multiple clusters.
+
+| <<collector,*Collector*>>
+| Collects and monitors container activities.
+| *1* on each node.
+
+| <<admission-controller,*Admission controller*>> (optional)
+| Interacts with Kubernetes API server and prevents creating workloads that do not adhere to security policies.
+| *1* for each cluster.
+|===
+
+
+.{product-title} components
+image::acs-architecture.svg[{product-title} components]
+
+[discrete]
+== Central
+
+**Central** is the main component of {product-title} and it is installed as a Kubernetes deployment.
```

Comment:
We use italics for emphasis or introducing a term like this, if you wouldn't mind making that adjustment throughout?